<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_buyBottle</class>
 <widget class="QWidget" name="page_buyBottle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_to_idle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_buyBottle_title">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>178</y>
     <width>698</width>
     <height>86</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_help_page">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>1732</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_help">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>1773</y>
     <width>125</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_no">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>1331</y>
     <width>294</width>
     <height>158</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yes_1">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>1331</y>
     <width>294</width>
     <height>158</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yes_2">
   <property name="geometry">
    <rect>
     <x>737</x>
     <y>1331</y>
     <width>294</width>
     <height>158</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_button_no_bg">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>1331</y>
     <width>294</width>
     <height>158</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_button_yes_bg_1">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>1331</y>
     <width>294</width>
     <height>158</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_button_yes_bg_2">
   <property name="geometry">
    <rect>
     <x>737</x>
     <y>1331</y>
     <width>294</width>
     <height>158</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_button_no_text_L">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>1361</y>
     <width>70</width>
     <height>52</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_button_no_text_S">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>1423</y>
     <width>144</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_button_yes_text_L_1">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>1361</y>
     <width>225</width>
     <height>52</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_button_yes_text_L_2">
   <property name="geometry">
    <rect>
     <x>767</x>
     <y>1361</y>
     <width>234</width>
     <height>52</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_button_yes_text_S_1">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>1423</y>
     <width>151</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_button_yes_text_S_2">
   <property name="geometry">
    <rect>
     <x>809</x>
     <y>1423</y>
     <width>151</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_buyBottle_title</zorder>
  <zorder>label_help</zorder>
  <zorder>pushButton_to_idle</zorder>
  <zorder>pushButton_help_page</zorder>
  <zorder>label_button_no_bg</zorder>
  <zorder>label_button_yes_bg_1</zorder>
  <zorder>label_button_yes_bg_2</zorder>
  <zorder>label_button_no_text_L</zorder>
  <zorder>label_button_no_text_S</zorder>
  <zorder>label_button_yes_text_L_1</zorder>
  <zorder>label_button_yes_text_L_2</zorder>
  <zorder>label_button_yes_text_S_1</zorder>
  <zorder>label_button_yes_text_S_2</zorder>
  <zorder>pushButton_no</zorder>
  <zorder>pushButton_yes_1</zorder>
  <zorder>pushButton_yes_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
